osd_cluster_admin/osd-cluster-notifications.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="osd-cluster-notifications"]
+= Cluster notifications
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: osd-cluster-notifications
+
+toc::[]
+
+include::snippets/managed-openshift-about-cluster-notifications.adoc[leveloffset=+0]
+
+[role="_additional-resources"]
+== Additional resources
+* xref:../osd_architecture/osd_policy/policy-process-security.adoc#notifications_policy-process-security[Customer responsibilities: Review and action cluster notifications]
+* xref:#cluster-notification-emails_osd-cluster-notifications[Cluster notification emails]
+* xref:#troubleshoot_osd-cluster-notifications[Troubleshooting: Cluster notifications]
```

Comment:
Even though this does work, technically should include the full path per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references).

Same for a few similar instances elsewhere in PR.